max_vel_x: 0.3
max_vel_y: 0.2
max_vel_th: 0.3

use_rviz_cam: true
rviz_cam_link: /rviz_cam

spacenav/max_val: 350.0
spacenav/min_val_th: 0.05 # threshold for filtering small values, should be in range <0,0.5>
